doc/user/content/installation/install-on-aws/_index.md

```diff
@@ -81,6 +81,13 @@ for evaluation purposes only. The modules deploy a sample infrastructure on AWS
 - Materialize Operator
 - Materialize instances (during subsequent runs after the Operator is running)
 
+- *Starting in v0.3.0 of Materialize on AWS Terraform*, AWS Load Balancer
```

Comment:
https://preview.materialize.com/materialize/31950/self-managed/v25.1/installation/install-on-aws/#set-up-aws-kubernetes-environment-and-install-materialize